docs/tutorials/course/mlp.md

```diff
@@ -0,0 +1,608 @@
+# Getting started
```

Comment:
Tutorials typically run live in our CI to produce their output, I'd recommend removing the hard-coded output in this tutorial.

Reply:
@zhanghang1989 Would this live in test_requirements, or does the CI use something else?

Reply:
OK

Reply:
Maybe we can call the tutorial name `Getting Started with Bayesian Optimization` or `Getting Started with Advanced HPO Algorithm`?